articles/videos/get-started/07_01-brand-how-it-works.md

```diff
@@ -0,0 +1,148 @@
+---
+description: Why your branding is important for your users and how it works with Auth0.
+classes: video-page
+---
+# Brand: How It Works
+
+Learn how branding works with Auth0 and how to add your company name and logo as well as choose colors for your pages.
```

Comment:
Awkward. Maybe "Learn how branding works with Auth0. Add your company name and logo, and choose colors for your pages."?